src/app/legend.ts

```diff
@@ -61,9 +71,9 @@ export class Legend {
       ) {
         currColors = this._colorMap.customColorMapWithMidpointByPercentage(
           colorMap,
-          this._settings.minValue,
+          this._world.newMin,
```

Comment:
These values should be stored in the `_settings` object instead, since these are in themselves settings. You should keep the `_settings.minValue`, but also add your `sliderMin` to that class and grab the values from there.